hedyorg/hedy
# [TRANSLATION] Add italian translations for the Parrot (Pappagallo) adventure

**Description**

**Adds** translations for the Parrot adventure in Italian

**How to test**

* launch the app: `python app.py`
* visit http://0.0.0.0:8080/hedy#parrot
* check that also the code on the left is translated
```
print Sono Hedy il pappagallo
ask Come ti chiami?
echo
echo
```
* go to http://0.0.0.0:8080/hedy/2#parrot
* check all the page is translated
* do the same check for:
  * http://0.0.0.0:8080/hedy/3#parrot
  * http://0.0.0.0:8080/hedy/5#parrot
 
**Checklist**
Done? Check if you have it all in place using this list:*
  
- [ ] Contains one of the PR categories in the name -> I didn't find the [TRANSLATION] tag
- [ ] Links to an existing issue or discussion  -> I've searched for "italian" in the issues but I've found none
